From 7de064b57cda7ca3d117c5faebc7636858dfc2f2 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 21 Apr 2015 10:34:31 +0200
Subject: [PATCH] ifcfg/write-ifcfg.sh: unset vlan

clear the vlan flag, if a previous interface in the loop
was a vlan interface

(cherry picked from commit 23446ef1e0a6eda79049a655cb1645691b129645)
---
 modules.d/45ifcfg/write-ifcfg.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/modules.d/45ifcfg/write-ifcfg.sh b/modules.d/45ifcfg/write-ifcfg.sh
index b2462da..ed66370 100755
--- a/modules.d/45ifcfg/write-ifcfg.sh
+++ b/modules.d/45ifcfg/write-ifcfg.sh
@@ -140,6 +140,8 @@ for netup in /tmp/net.*.did-setup ; do
     unset macaddr
     unset slave
     unset ethname
+    unset vlan
+
     [ -e /tmp/bond.${netif}.info ] && . /tmp/bond.${netif}.info
     [ -e /tmp/team.${netif}.info ] && . /tmp/team.${netif}.info
 
